eoscostarica/eosio-spend-explorer
# feat(webapp): add table and graph for income

### Add table and graph for income

### What does this PR do?

- Resolve #47 
- Resolve #51 

### Steps to test

1. run the project
2. go to the sidebar and click on income
3. See the view

#### CheckList

- [] Follow proper Markdown format
- [] The content is adequate
- [] The content is available in both english and spanish
- [] I Ran a spell check
